src/App.js

```diff
@@ -25,33 +31,72 @@ const App = () => {
     setData(responseJson)
   }
 
-  useEffect(() => {
-    fetchImageData()
-    fetchFavData()
-  }, [])
+  const fetchCartData = async () => {
+    const response = await fetch('http://localhost:3001/api/v1/cart')
+    const responseJson = await response.json()
+    setCart(responseJson)
+  }
+
+  const addToCart = async (newData) => {
+    const config = {
+      method: 'POST',
+      headers: {
+        'Accept': 'application/json',
+        'Content-Type': 'application/json'
+      },
+      body: JSON.stringify(newData)
+    };
+    try {
+    const fetchResponse = await fetch('http://localhost:3001/api/v1/cart', config)
+    const json = await fetchResponse.json()
+    fetchCartData()
+    console.log(json)
+    } catch (err) {
+      console.log(err)
+      return err;
+    }
+  }
 
   const addToFavorites = async (newData) => {
-    await fetch('http://localhost:3001/api/v1/favorites', {
+    const config = {
       method: 'POST',
       headers: {
+        'Accept': 'application/json',
         'Content-Type': 'application/json'
       },
       body: JSON.stringify(newData)
-    })
-        await (res => res.json())
-        await fetchFavData()
-        await fetchImageData()
-        .catch(err => console.log(err))
+    };
+    try {
+    const fetchResponse = await fetch('http://localhost:3001/api/v1/favorites', config)
+    const json = await fetchResponse.json()
+    fetchImageData()
+    fetchFavData()
+    console.log(json)
+    } catch (err) {
+      console.log(err)
+      return err;
+    }
   }
 
   const deleteFromFavorites = async (newData) => {
-    await fetch(`http://localhost:3001/api/v1/favorites/${newData.id}`, {
-      method: 'DELETE'
-    })
-      await ((res) => res.json())
-      await fetchFavData()
-      await fetchImageData()
-      .catch(err => console.log(err))
+    const config = {
+      method: 'Delete',
```

Comment:
Didn't know method names didn't need to be capitalized? If it works that's good to know for future troubleshooting